### ISSUE TITLE
Bower integration

### ISSUE THREAD

**Issue by fuhlig** [csshugs/patternlab-scaffolding#1]
The current bower integration lacks in real frontend dependency management, as script files have to be manually downloaded or copied from the `bower_components` directory in order to use - same goes for updating. This manual process is bowers' job to automate.

To inject the dependencies into the source code, [grunt-wiredep](https://github.com/stephenplusplus/grunt-wiredep) can be used.
Only packages installed declared as dependencies (`--save`) are injected.
## Grunt

Watch-task on `bower.json` to run `wiredep` after installing packages.
## Scripts

Scripts to be injected into `foot.mustache` via

```
<!-- bower:js -->
<!-- endbower -->
```

Concat, minify and uglify should be done similar to typical yeoman generators using `grunt-usemin` and cache-busting revisions (_might not be needed due to PatternLab's / CMS / shop / ... cache-busting_)
## Styles

From my experience not necessary to be injected into `<head>` due to Sass. Style plugins usually need modifications so current workflow (copy into style directory and manual @import) works.
## Patternlab

`/bower_components`needs to be resolved proberly, similar to `grunt-contrib-sass`

```
connect().use('/bower_components', connect.static('./bower_components'))
```

**Comment by csshugs** [comment]
@fuhlig Maybe we can discuss this face to face? :beer:

**Comment by fuhlig** [comment]
The previously proposed concept can be accomplished with a combination of [grunt-injector](https://github.com/klei/grunt-injector) and [grunt-bowercopy](https://github.com/timmywil/grunt-bowercopy).
More details and concept will come soon

**Comment by csshugs** [comment]
closed with v0.7.0.
